From 3263bb13f92a8194d77c85b65415fac3a9b63e15 Mon Sep 17 00:00:00 2001
From: Hyun Kwon <hyun.kwon@xilinx.com>
Date: Tue, 18 Mar 2014 09:18:19 -0700
Subject: [PATCH 463/827] media: xilinx: dma: Fix alignments of xvip_dma_fops definition

This patch comes from:
  https://github.com/Xilinx/linux-xlnx.git

Fix alignments of xvip_dma_fops definition.

Signed-off-by: Hyun Kwon <hyunk@xilinx.com>
Acked-by: Laurent Pinchart <laurent.pinchart@ideasonboard.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
(cherry picked from commit 06fda8395981bff3f134d0a13d6105b0410634af)
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 drivers/media/platform/xilinx/xilinx-dma.c | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/drivers/media/platform/xilinx/xilinx-dma.c b/drivers/media/platform/xilinx/xilinx-dma.c
index 3aa5c4b7..98c2a49 100644
--- a/drivers/media/platform/xilinx/xilinx-dma.c
+++ b/drivers/media/platform/xilinx/xilinx-dma.c
@@ -640,12 +640,12 @@ static int xvip_dma_mmap(struct file *file, struct vm_area_struct *vma)
 }
 
 static struct v4l2_file_operations xvip_dma_fops = {
-	.owner = THIS_MODULE,
-	.unlocked_ioctl = video_ioctl2,
-	.open = xvip_dma_open,
-	.release = xvip_dma_release,
-	.poll = xvip_dma_poll,
-	.mmap = xvip_dma_mmap,
+	.owner		= THIS_MODULE,
+	.unlocked_ioctl	= video_ioctl2,
+	.open		= xvip_dma_open,
+	.release	= xvip_dma_release,
+	.poll		= xvip_dma_poll,
+	.mmap		= xvip_dma_mmap,
 };
 
 /* -----------------------------------------------------------------------------
-- 
2.9.3

